static/js/compose.js

```diff
@@ -325,6 +325,67 @@ exports.send_message = function send_message(request) {
     }
 };
 
+function is_scheduled_message(message_content) {
+    return /^\/remind/.test(message_content);
+}
+
+function patch_request_for_scheduling(request) {
+    var new_request = request;
+    var raw_message = request.content.split('\n');
+    var command_line = raw_message[0];
+    var message = raw_message.slice(1).join('\n');
+    var deliver_at = command_line.slice(8);
+
+    if (message.trim() === '' || deliver_at.trim() === '' || command_line.slice(7,8) !== ' ') {
```

Comment:
The 7,8 here seems very fragile.  You should just check with actual string values.